release

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env ruby
+require 'fog'
+require 'dotenv'
+require 'json'
+require 'digest'
+require 'set'
+require 'git'
+
+Dotenv.load
+
+ALL_RELEASES = "releases/all.json"
+LATEST_RELEASE = "releases/latest.json"
+GIT_DIR = File.expand_path(__FILE__ + "/..")
```

Comment:
Yeah. [This suggestion](https://github.com/Shopify/themekit/pull/84#discussion_r39419982) will come in handy to avoid duplicating them in multiple classes.